content/neuvector-prime/azure/contents.lr

```diff
@@ -0,0 +1,476 @@
+kube-system   konnectivity-agent                                        2/2     2            2           19m
+kube-system   metrics-server                                            2/2     2            2           19m
+neuvector     neuvector-controller-pod                                  3/3     3            3           15m
+neuvector     neuvector-csp-pod                                         1/1     1            1           15m
+neuvector     neuvector-manager-pod                                     1/1     1            1           15m
+neuvector     neuvector-scanner-pod                                     3/3     3            3           15m
+```
+
+
+Jobs and Pods:
+
+Check that pods or jobs have status Running/Completed
+
+
+```
+kubectl get pods --all-namespaces
+```
+
+
+if a pod is not in Running state, you can dig into the root cause by running:
+      
+Describe pod
+
+
+```
+kubectl describe pod <pod name> -n <namespaces>
+```
+
+Pod container logs
+
+
+```
+kubectl logs <pod name> -n <namespaces>
+```
+
+Describe job
+
+
+```
+kubectl describe job <job name> -n <namespaces>
+```
+
+Logs from the containers of pods of the job
+
+```
+kubectl logs -l job-name=<job name> -n <namespaces>
+```
+
+###NeuVector Usage Record Not found: 
+
+Error:
+
+```
+Error from server (NotFound): cspadapterusagerecords.susecloud.net "neuvector-usage" not found"
+ Check Configuration, Retrieve generated configuration csp-config
+```
+    
+
+Solution:
+
+```
+kubectl get cm -n neuvector csp-config -oyaml
+```
+
+
+if a configuration is not listed, you can dig into the root cause by checking the pod status and log (Refer Jobs and Pods section).
+
+
+###Multiple extensions of same type: 
+
+Error:
+
+```
+Multiple extensions of same type is not allowed at this scope. (Code: ValidationFailed)"
+```
+
+Solution:
+
+AKS cluster already has the extension with the same type. Uninstall the extension and re-deploy with the same cluster.
+
+   !["Uninstall extension"](./uninstall.png)
+
+###Resource already existing in your cluster: 
+
+Error:
+
+```
+Helm installation failed : Resource already existing in your cluster : Recommendation Manually delete the resource(s) that currently exist in your cluster and try installation again. To delete these resources run the following commands: kubectl delete <resource type> -n <resource namespace> <resource name> : InnerError [rendered manifests contain a resource that already exists. Unable to continue with install: ServiceAccount "csp" in namespace "neuvector" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-name" must equal "test-nv2-reinstall": current value is "testnv2-plan"]
+```
+
+Solution:
+
+AKS cluster already has the extension. Uninstall the extension as it suggested in the Error by deleting the resource via the kubectl command. or Uninstall the extension in Azure Console and re-deploy with the same cluster. 
```

Comment:
Uninstall the extension as suggested in the error

change "or" to "Alternately,"